src/components/nodes/eventBasedGateway/index.js

```diff
@@ -1,16 +1,19 @@
 import component from './eventBasedGateway.vue';
 import advancedAccordionConfig from '@/components/inspectors/advancedAccordionConfig';
+import defaultNames from '@/components/nodes/gateway/defaultNames';
+
+const id = 'processmaker-modeler-event-based-gateway';
 
 export default {
-  id: 'processmaker-modeler-event-based-gateway',
+  id,
   component,
   bpmnType: 'bpmn:EventBasedGateway',
   control: false,
   category: 'BPMN',
-  label: 'Event-Based Gateway',
```

Comment:
This one was either `Event-Based Gateway` or `Event Based Gateway`. I switched it to the crown item.